spec.emu

```diff
@@ -2,19 +2,51 @@
 <meta charset="utf8">
 <script src="ecmarkup.js"></script>
 <link rel="stylesheet" href="ecmarkup.css">
-<title>import assertions</title>
+<title>Import Assertions</title>
 <pre class=metadata>
-  title: Import assertions
+  title: Import Assertions
   status: proposal
   stage: 2
   location: https://github.com/tc39/proposal-import-assertions
   copyright: false
   contributors: Sven Sauleau, Myles Borins, Daniel Ehrenberg, Daniel Clark
 </pre>
+
+<script>
```

Comment:
These `<script>`/`<style>` tags are needed to get this nice rendering:

<table><tr><td>

![](https://user-images.githubusercontent.com/7000710/222485031-69e061c8-8835-475e-ba3d-f641e923b2c3.png)

</td><td>

![](https://user-images.githubusercontent.com/7000710/222485264-0fa77da2-6816-44ba-877b-096d94bffede.png)

</td></tr></table>